### PR TITLE
Nunez deploy docs 3

### PR DESCRIPTION
**Pull request by stellardave**
Full edit of Deploy section.

### CODE REVIEW

File: docs/3.0rc/deploy/dynamic-infra/deploy-docker.mdx [hunk 1]
```diff
@@ -155,16 +169,23 @@ Check that your build worked by running a container from your new image.
     ```
 
     After running the above command, the container should start up and serve the flow within the container.
```
Comment (discdiver) [inline docs/3.0rc/deploy/dynamic-infra/deploy-docker.mdx:171]:
```suggestion
    After running the above command, the container should start and the flow should run within the container.
```

File: docs/3.0rc/deploy/dynamic-infra/deploy-docker.mdx [hunk 2]
```diff
@@ -155,16 +169,23 @@ Check that your build worked by running a container from your new image.
...
+This guide simulates a remote environment by using Kubernetes locally with Docker Desktop. 
+Use the instructions provided by Docker to 
+[set up Kubernetes locally.](https://docs.docker.com/desktop/kubernetes/)
 
 ### Create a Kubernetes deployment manifest
 
-To ensure the process serving your flow is always running, create a [Kubernetes deployment](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/). If your flow's container ever crashes, Kubernetes will automatically restart it, ensuring that you don't miss any scheduled runs.
+To ensure the process serving your flow is always running, 
+create a [Kubernetes deployment](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/). 
+If your flow's container ever crashes, Kubernetes automatically restarts it,
```
Comment (discdiver) [inline docs/3.0rc/deploy/dynamic-infra/deploy-docker.mdx:187]:
```suggestion
If your flow's container crashes, Kubernetes automatically restarts it, 
```

File: docs/3.0rc/deploy/dynamic-infra/deploy-docker.mdx [hunk 3]
```diff
@@ -202,11 +224,16 @@ Add the following content to your `deployment-manifest.yaml` file:
...
             imagePullPolicy: Never
     ```
+<Tip>
+**Keep your API key secret**
 
-    !!!tip "Keep your API key secret"
-          In the above manifest, you are passing in the Prefect API URL and API key as environment variables. This approach is simple, but it is not secure. If you are deploying your flow to a remote cluster, you should use a [Kubernetes secret](https://kubernetes.io/docs/concepts/configuration/secret/) to store your API key.
-
-=== "Self-hosted"
+In the above manifest, you are passing in the Prefect API URL and API key as environment variables.
```
Comment (discdiver) [inline docs/3.0rc/deploy/dynamic-infra/deploy-docker.mdx:230]:
```suggestion
In the above manifest, you are passing the Prefect API URL and API key as environment variables. 
```

File: docs/3.0rc/deploy/dynamic-infra/deploy-docker.mdx [hunk 4]
```diff
@@ -280,17 +317,29 @@ Additionally, we have "convenience" tags which are updated with each release to
 
 For example, when release `2.11.5` is published:
```
Comment (discdiver) [inline docs/3.0rc/deploy/dynamic-infra/deploy-docker.mdx:318]:
```suggestion
For example, when release `3.1.1` is published:
```

File: docs/3.0rc/deploy/dynamic-infra/deploy-docker.mdx [hunk 5]
```diff
@@ -280,17 +317,29 @@ Additionally, we have "convenience" tags which are updated with each release to
...
-4. For users that want to be on the latest `2.1.x` release and receiving patch updates, we update a tag without the patch version to this release. For example, `prefect.2.1-python3.10`.
-5. For users that want to be on the latest `2.x.y` release and receiving minor version updates, we update a tag without the minor or patch version to this release. For example, `prefect.2-python3.10`.
-6. For users who want the latest `2.x.y` release without specifying a Python version, we update `2-latest` to the image for our highest supported Python version, which in this case is equivalent to `prefect:2.1.1-python3.10`.
-
-!!! tip "Choose image versions carefully"
-    It's a good practice to use Docker images with specific Prefect versions in production.
-
-    Use care when employing images that automatically update to new versions (such as `prefecthq/prefect:2-python3.11` or `prefecthq/prefect:3-latest`).
+1. Images with the release packaged are built for each supported Python version 
+(3.8, 3.9, 3.10, 3.11) with both standard Python and Conda.
```
Comment (discdiver) [inline docs/3.0rc/deploy/dynamic-infra/deploy-docker.mdx:321]:
```suggestion
(3.9, 3.10, 3.11, 3.12) with both standard Python and Conda.
```

File: docs/3.0rc/deploy/dynamic-infra/deploy-docker.mdx [hunk 6]
```diff
@@ -14,11 +17,16 @@ This guide covers how to:
 - Deploy and run your Docker image on a Kubernetes cluster.
 - Look at the Prefect-maintained Docker images and discuss options for use.
```
[the remark this answers is not in the document]
Comment (discdiver) [reply docs/3.0rc/deploy/dynamic-infra/deploy-docker.mdx:18]: The Docker image selection strategy section also appeals (and will mostly be used by folks who are using work-pool based deployments. Should we break it out into its own page in the Resources section. cc: @billpalombi

File: docs/3.0rc/deploy/dynamic-infra/retrieve-workflow-code.mdx [hunk 7]
```diff
@@ -38,43 +43,53 @@ Would you like your workers to pull your flow code from its remote repository wh
...
 In this example, the git repository is hosted on GitHub.
 If you are using Bitbucket or GitLab, the URL will match your provider.
-If the repository is public, enter "n" and you are on your way.
+If the repository is public, enter "n".
 
-If the repository is private, you can enter a token to access your private repository. This token will be saved in an encrypted Prefect Secret block.
+If the repository is private, you can enter a token to access your private repository. 
+This token is saved to an encrypted Prefect Secret block.
 
 <div class="terminal">
```
[the remark this answers is not in the document]
Comment (discdiver) [reply docs/3.0rc/deploy/dynamic-infra/retrieve-workflow-code.mdx:51]: Probably just easier to find and replace these divs. They don't really hurt. I can clean them up later.

File: docs/3.0rc/develop/observe-workflows/manage-states.mdx [hunk 8]
```diff
@@ -48,7 +48,7 @@ The full list of states and state types includes:
 | <span class="no-wrap">AwaitingRetry</span> | SCHEDULED | No | The run did not complete successfully because of a code issue and had remaining retry attempts. |
 | Pending | PENDING | No | The run has been submitted to run, but is waiting on necessary preconditions to be satisfied. |
 | Running | RUNNING | No | The run code is currently executing. |
-| Retrying | RUNNING | No | The run code is currently executing after previously not complete successfully. |
+| Retrying | RUNNING | No | The run code is currently executing after previously not completed successfully. |
```
Comment (discdiver) [inline docs/3.0rc/develop/observe-workflows/manage-states.mdx:51]:
```suggestion
| Retrying | RUNNING | No | The run code is currently executing after previously not completing successfully. |
```

**Review by discdiver — COMMENTED** [review]
Lot's of good stuff! Thank you.

File: docs/3.0rc/deploy/dynamic-infra/deploy-docker.mdx [hunk 9]
```diff
@@ -232,39 +259,49 @@ Add the following content to your `deployment-manifest.yaml` file:
...
-Once the deployment has successfully started, you can check the logs of your flow container by running the following:
+Once the deployment successfully starts, check the logs of your flow container 
+by running the following:
 
 ```bash
 kubectl logs -l flow=get-repo-info
 ```
 
-Now that you're serving your flow in your cluster, you can trigger a flow run by running:
+You're now serving your flow in your cluster. To trigger a flow, run:
```
Comment (billpalombi) [inline docs/3.0rc/deploy/dynamic-infra/deploy-docker.mdx:297]:
```suggestion
You're now serving your flow in your cluster. To trigger a flow run, use:
```

File: docs/3.0rc/deploy/dynamic-infra/deploy-docker.mdx [hunk 10]
```diff
@@ -232,39 +259,49 @@ Add the following content to your `deployment-manifest.yaml` file:
...
 
-Now that you have a deployment manifest, you can deploy your flow to the cluster by running:
+You now have a deployment manifest. To deploy your flow to the cluster, run:
 
 ```bash
 kubectl apply -f deployment-manifest.yaml
 ```
 
-Monitor the status of your Kubernetes deployment by running:
+To monitor the status of your Kubernetes deployment, run:
```
Comment (billpalombi) [inline docs/3.0rc/deploy/dynamic-infra/deploy-docker.mdx:284]:
```suggestion
To monitor the status of your Kubernetes deployment, use:
```

File: docs/3.0rc/deploy/dynamic-infra/deploy-docker.mdx [hunk 11]
```diff
@@ -232,39 +259,49 @@ Add the following content to your `deployment-manifest.yaml` file:
...
-This manifest defines how your image will run when deployed in your Kubernetes cluster. Note that you will run a single replica of your flow container. If you want to run multiple replicas of your flow container to keep up with an active schedule, or because your flow is resource-intensive, you can increase the `replicas` value.
+This manifest defines how your image runs when deployed in your Kubernetes cluster. 
+Note that you will run a single replica of your flow container. 
+If you want to run multiple replicas of your flow container to keep up with an active schedule, 
+or because your flow is resource-intensive, you can increase the `replicas` value.
 
 ### Deploy your flow to the cluster
 
-Now that you have a deployment manifest, you can deploy your flow to the cluster by running:
+You now have a deployment manifest. To deploy your flow to the cluster, run:
```
Comment (billpalombi) [inline docs/3.0rc/deploy/dynamic-infra/deploy-docker.mdx:278]:
```suggestion
You now have a deployment manifest. To deploy your flow to the cluster, use:
```

File: docs/3.0rc/deploy/dynamic-infra/deploy-kubernetes.mdx [hunk 12]
```diff
@@ -354,19 +395,23 @@ if __name__ == "__main__":
 ```
 
 Run the flow locally with `python hello.py` to verify that it works.
-Note that we use the `tags` context manager to tag the flow run as `local`.
+We use the `tags` context manager to tag the flow run as `local`.
```
Comment (billpalombi) [inline docs/3.0rc/deploy/dynamic-infra/deploy-kubernetes.mdx:398]:
```suggestion
Use the `tags` context manager to tag the flow run as `local`.
```

File: docs/3.0rc/deploy/dynamic-infra/push-runs-remote.mdx [hunk 13]
```diff
@@ -159,28 +156,32 @@ if __name__ == "__main__":
...
 
-
 ```bash
 python repo_info.py
 ```
-
-
-Prefect builds a custom Docker image containing your workflow code that the worker can use to dynamically spawn Docker containers whenever this workflow needs to run.
+    
+Prefect builds a custom Docker image containing your workflow code. The worker can use it to dynamically spawn Docker
```
Comment (billpalombi) [inline docs/3.0rc/deploy/dynamic-infra/push-runs-remote.mdx:176]:
```suggestion
Prefect builds a custom Docker image containing your flow code. The worker can use it to dynamically spawn Docker 
```

**Review by billpalombi — COMMENTED** [review]
Looks good, thanks!

**Review by billpalombi — APPROVED** [review]
lgtm

**Pull request merged**